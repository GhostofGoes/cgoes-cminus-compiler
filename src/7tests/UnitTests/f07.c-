int cat()
{
    return 93;
}

main()
{
    output(cat());
    outnl();
}
