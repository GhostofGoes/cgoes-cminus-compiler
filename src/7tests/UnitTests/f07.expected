Number of warnings: 0
Number of errors: 0
------------------------------------

Memory Configuration: Data Addresses: 0-9999  Instruction Addresses: 0-9999
Loading file: f07.tm
93 
Status: Halted
Bye.
